src/simd_128.h

```diff
@@ -252,6 +252,35 @@ add_two_bufs(uint32_t* src, uint32_t* dest, size_t len, uint32_t card = F4)
     }
 }
 
+inline void sub_two_bufs(
+    uint32_t* bufa,
+    uint32_t* bufb,
+    uint32_t* res,
+    size_t len,
+    uint32_t card = F4)
+{
+    __uint128_t* _bufa = static_cast<__uint128_t*>(static_cast<void*>(bufa));
+    __uint128_t* _bufb = static_cast<__uint128_t*>(static_cast<void*>(bufb));
+    __uint128_t* _res = static_cast<__uint128_t*>(static_cast<void*>(res));
+    size_t _len = len / 4;
```

Comment:
Maybe we could replace 4 (which is "hardcoded") by something like `sizeof(bufb[0])/sizeof(bufb[0]))` or `sizeof(__uint128_t)/sizeof(uint32_t)` and store the result in a named constant that we can use hither and yon.

Reply:
Some functions (as this one) would be generalized using template. I will use this calculation in the generalized functions.